pdns/recursordist/docs/changelog/4.2.rst

```diff
@@ -1,5 +1,44 @@
 Changelogs for 4.2.x
 ====================
+.. changelog::
+  :version: 4.2.4
+  :released: 17th of July 2020
+
+  .. change::
+    :tags: XXXXXX
```

Comment:
I guess that's still a work in progress since all tags are missing for 4.2.x? :-)